tests/test_salami.py

```diff
@@ -11,6 +11,10 @@
 def mock_validate(mocker):
     return mocker.patch.object(salami, 'validate')
 
+@pytest.fixture
+def mock_load_sections(mocker):
+  return mocker.patch.object(salami, '_load_sections')
```

Comment:
I thought it was `load_sections` not `_load_sections`?

Reply:
Disregard, looking in the code I see now that it is `_load_sections` - that's awesome, looks like we fixed a bad test